setup/edge-pc-local-setup.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+cat << "EOF"
+    __ ____________    ____           __        ____      __  _
+   / //_/__  / ___/   /  _/___  _____/ /_____ _/ / /___ _/ /_(_)___  ____
+  / ,<   /_ <\__ \    / // __ \/ ___/ __/ __ `/ / / __ `/ __/ / __ \/ __ \
+ / /| |___/ /__/ /  _/ // / / (__  ) /_/ /_/ / / / /_/ / /_/ / /_/ / / / /
+/_/ |_/____/____/  /___/_/ /_/____/\__/\__,_/_/_/\__,_/\__/_/\____/_/ /_/
+
+EOF
+
+#######################################
+#   STEP 0: CHECK SUDO PRIVILEGES     #
+#######################################
+
+if [ "$EUID" -ne 0 ]
+  then echo "This script need sudo privileges, please run: sudo ./$(basename $0)"
+  exit
+fi
+
+echo "K3S cluster setup started"
+echo ""
+
+#################################################
+#   STEP 1: MANAGEMENT USER CREATION            #
+#################################################
+
+# Create mgmt user
+echo -n "Management user creation..."
+
+adduser --gecos "Management user" --disabled-password mgmt >/dev/null
```

Comment:
Ok, that's something I missed. 

Anyways, we should not expose credentials into scripts as this would create security leaks.

A possibility that comes into my mind could be using Ansible Vault, but any feedback and possible alternatives are more than welcome.

Let's further investigate on this.

Thanks!

Reply:
This is a really good idea, it would require the usage of Ansible variables which can be passed as command line parameters when launching `ansible-playbook` command.

However, this could not be the best option if automation of these Ansible scripts is required: a user is needed to prompt the desired password.

That's why Ansible Vault (or similiar way to automate the task) is more suitable for this use-case.